packages/iframe-stamper/src/index.ts

```diff
@@ -145,29 +149,29 @@ export class IframeStamper {
 
     // This is populated once the iframe is ready. Call `.init()` to kick off DOM insertion!
     this.iframePublicKey = null;
+
+    this.messageChannel = new MessageChannel();
   }
 
   /**
    * Inserts the iframe on the page and returns a promise resolving to the iframe's public key
    */
   async init(): Promise<string> {
     this.container.appendChild(this.iframe);
+    this.iframe.addEventListener("load", () => {
+      // Send a message to the iframe to initialize the message channel
+      this.iframe.contentWindow?.postMessage("init", this.iframeOrigin, [
```

Comment:
Another comment here. Maybe this message should be called "initChannel" or "INIT_CHANNEL" just so it's clear?

Reply:
Updated this to be `TURNKEY_INIT_MESSAGE_CHANNEL`